docs/source/developer_guide/core/modifying_the_schema.rst

```diff
@@ -3,220 +3,192 @@ Modifying the schema
-
-    ./alembic_manage.py --help
-
-you will get a full list of the available arguments that you can pass and
-commands. Briefly, the available commands are:
-
-* **upgrade** This command allows you to upgrade to the later version. For the
-  moment, you can only upgrade to the latest version.
-* **downgrade** This command allows you to downgrade the version of your
-  database. For the moment, you can only downgrade to the base version.
-* **history** This command lists the available migrations in chronological
-  order.
+ 1. Make all the necessary changes to the model than you would like to modify located in the ``aiida/backends/sqlalchemy/models`` directory.
+
+ 2. Create new migration file by going to ``aiida/backends/sqlalchemy`` and executing::
+
+        python aiida/backends/sqlalchemy/manage.py revision "This is the description for the next revision"
+
+    This will create a new migration file in ``aiida/backends/sqlalchemy/migrations/versions`` whose names begins with an automatically generated hash and the provided message for this new migration.
+    Modify the migration message to accurately describe the purpose of the migration.
+
+ 3. Have a look at the generated migration file and ensure that migration is correct.
+    The file should contain automatically generated hashes that point to the previous and to the current revision::
+
+        revision = 'e72ad251bcdb'
+        down_revision = 'b8b23ddefad4'
+
+    Also ``upgrade()`` and ``downgrade()`` function definitions should be present in the file::
+
+        def upgrade():
+            # some upgrage operations
+        def downgrade():
+            # some downgrade operations
+
+    If you want to add some changes that affect the content of the database -- you should do it "manually" by adding some sql commands that will run directly on your database.
+    Learn the following example and adapt it for your needs::
+
+        from sqlalchemy.sql import text
+
+        forward_sql = [
+            """UPDATE db_dbgroup SET type_string = 'auto.import' WHERE type_string = 'aiida.import';""",
+            """UPDATE db_dbgroup SET type_string = 'auto.run' WHERE type_string = 'autogroup.run';""",
+        ]
+
+        reverse_sql = [
+            """UPDATE db_dbgroup SET type_string = 'aiida.import' WHERE type_string = 'auto.import';""",
+            """UPDATE db_dbgroup SET type_string = 'autogroup.run' WHERE type_string = 'auto.run';""",
+            ]
+
+        def upgrade():
+            conn = op.get_bind()
+            statement = text('\n'.join(forward_sql))
+            conn.execute(statement)
+        def downgrade():
+            conn = op.get_bind()
+            statement = text('\n'.join(reverse_sql))
+            conn.execute(statement)
+
+    If you want to learn more about the migration operations, you can have a look at the Alembic documentation.
+
+ 4. Your database will be automatically migrated to the latest revision as soon as you run your first verdi command.
```

Comment:
Is this still true?

Reply:
Nope, you are right. It now follows the same paradigm as Django and simply prompts a command to be executed

Reply:
I also realize now that the Django section still instructs to use the Django manage tool directly, i.e. `python aiida/backends/djsite/manage.py migrate`. I think people, even developers, should use `verdi database migrate` instead. Should I homogenize the two sections to reflect this?

Reply:
Sure, if you can it would be great!